src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandlerTest.java

```diff
@@ -173,5 +176,20 @@ public void testValidateRespondentPaymentDate() {
             assertEquals("Validation error", response.getErrors().get(0));
         }
 
+        @Test
+        public void testValidateRepaymentDate() {
+            CaseData caseData = CaseDataBuilder.builder().generateRepaymentDateForAdmitPartResponse().build();
+            CallbackParams params = callbackParamsOf(caseData, MID, "validate-repayment-plan", "DEFENDANT_RESPONSE_SPEC");
+            when(dateValidator.validateFuturePaymentDate(any())).thenReturn(List.of("Validation error"));
+
+            AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse) handler
+                .handle(params);
+
+            List<String> expectedErrorArray = new ArrayList<>();
+            expectedErrorArray.add("Date must be within the next 12 months");
```

Comment:
expectedErrorArray is unused in this test. Please check and remove

Reply:
okay, removing it

Reply:
changes incorporated